### PR TITLE
minor corrections, line breaks, capitalization, expand abbreviations

### PR DESCRIPTION
**Pull request by cjschroder**
allrightythen, the final Kanidm book review! Like the others this is a light proofread, fixing punctuation, capitalization, spelling, inserting linebreaks to make the source files a little easier to read, and such. Tis a most excellent book for a promising product, you done good!

- [ ] cargo fmt has been run
- [ ] cargo clippy has been run
- [ ] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: DEVELOPER_README.md [hunk 1]
```diff
@@ -22,37 +23,43 @@ The MSRV is specified in the package `Cargo.toml` files.
...
-You will need [rustup](https://rustup.rs/) to install a rust toolchain.
+You will need [rustup](https://rustup.rs/) to install a Rust toolchain.
 
-If you plan to work on the web-ui, you may also need npm for setting up some parts.
+If you plan to work on the Web user interface, you may also need npm for setting 
+up some parts.
 
     brew install npm
+    
+<!-- Is this working in the Web UI, instead of on the command line, or working
```
Comment (yaleman) [inline DEVELOPER_README.md:33]: working *on*, making changes to it etc 😄

File: DEVELOPER_README.md [hunk 2]
```diff
@@ -22,37 +23,43 @@ The MSRV is specified in the package `Cargo.toml` files.
...
     brew install npm
+    
+<!-- Is this working in the Web UI, instead of on the command line, or working 
+    on the Web UI, to change or customize it? -->    
 
+    
 #### SUSE
 
-You will need [rustup](https://rustup.rs/) to install a rust toolchain. If you're using the Tumbleweed release, it's packaged in `zypper`.
+You will need [rustup](https://rustup.rs/) to install a rust toolchain. If you're
```
Comment (yaleman) [inline DEVELOPER_README.md:39]: `Rust` here

File: DEVELOPER_README.md [hunk 3]
```diff
@@ -160,25 +178,28 @@ In a new terminal, you can now build and run the client tools with:
...
-Then you are able to build the UI.
+Then you are able to build the UI:
 
     cd kanidmd_web_ui/
     ./build_wasm.sh
 
 The "developer" profile for kanidmd will automatically use the pkg output in this folder.
 
-Setting different developer profiles while building is done by setting the environment variable KANIDM_BUILD_PROFILE to one of the bare filename of the TOML files in `/profiles`. 
+Setting different developer profiles while building is done by setting the environment v
```
Comment (yaleman) [inline DEVELOPER_README.md:197]: Line break broke `variable` here 😄

**Review by yaleman — CHANGES_REQUESTED** [review]
Thanks 🥳

File: DEVELOPER_README.md (hunk 1 above)
Comment (yaleman) [reply DEVELOPER_README.md:33]: Can confirm, `npm` is no longer needed for WASM building.

File: DEVELOPER_README.md (hunk 3 above)
Comment (cjschroder) [reply DEVELOPER_README.md:197]: sheesh so picky

Comment (cjschroder) [reply DEVELOPER_README.md:197]: 😁

**Comment by yaleman** [comment]
Woooooooooo, thanks again! 🥳

**Pull request merged**